### PR TITLE
docs: improved the en version of create_tb

### PR DESCRIPTION
**Pull request by michelle-qinqin**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**
improved some statements
fixed the table

### CODE REVIEW

File: docs/en/reference/sql/ddl/CREATE_TABLE_STATEMENT.md [hunk 1]
```diff
@@ -1,40 +1,35 @@
 # CREATE TABLE
+ The `CREATE TABLE` statement is used to create a table. The table name must be unique in one database. If the table with the same name in a database is created repeatedly, an error will occur.
```
Comment (lumianph) [inline docs/en/reference/sql/ddl/CREATE_TABLE_STATEMENT.md]: you can just remove the third sentence " If the table with the same name in a database is created repeatedly, an error will occur."

File: docs/en/reference/sql/ddl/CREATE_TABLE_STATEMENT.md [hunk 2]
```diff
@@ -1,40 +1,35 @@
...
 TableElement ::=
-    ColumnDef
-|   ColumnIndex
+    ColumnDef | ColumnIndex
 ```
 
- The `CREATE TABLE` statement is used to create a table. The table name must be unique if it's in the same database. If the table with the same name is created repeatedly, an error will occur.
 
-The `table_element` list needs to be defined in the table creation statement. `table_element` is divided into column description `ColumnDef` and `Constraint`. OpenMLDB requires at least one ColumnDef in the `table_element` list.
+The `table_element` list needs to be defined in the `CREATE TABLE` statement. `table_element` is divided into column description `ColumnDef` and `Constraint`. OpenMLDB requires at least one `ColumnDef` in the `table_element` list.
```
Comment (lumianph) [inline docs/en/reference/sql/ddl/CREATE_TABLE_STATEMENT.md]: what is table_element?
is it that TableElementList?

**Review by lumianph — CHANGES_REQUESTED** [review]
the structure of this doc is a bit messy, discuss with you later to revise the structure

File: docs/en/reference/sql/ddl/CREATE_TABLE_STATEMENT.md (hunk 2 above)
Comment (michelle-qinqin) [reply docs/en/reference/sql/ddl/CREATE_TABLE_STATEMENT.md]: yes, `table_element` list is equal to TableElementList

Comment (lumianph) [reply docs/en/reference/sql/ddl/CREATE_TABLE_STATEMENT.md]: there is no context about table_element, should change to TableElementList

Comment (michelle-qinqin) [reply docs/en/reference/sql/ddl/CREATE_TABLE_STATEMENT.md]: ok
I will change it

**Comment by michelle-qinqin** [comment]
@lumianph 标题结构和磁盘表标题均已修改

File: docs/en/reference/sql/ddl/CREATE_TABLE_STATEMENT.md [hunk 3]
```diff
@@ -1,40 +1,34 @@
...
 TableElement ::=
-    ColumnDef
-|   ColumnIndex
+    ColumnDef | ColumnIndex
 ```
 
- The `CREATE TABLE` statement is used to create a table. The table name must be unique if it's in the same database. If the table with the same name is created repeatedly, an error will occur.
 
-The `table_element` list needs to be defined in the table creation statement. `table_element` is divided into column description `ColumnDef` and `Constraint`. OpenMLDB requires at least one ColumnDef in the `table_element` list.
+The `TableElementList` needs to be defined in the `CREATE TABLE` statement. `TableElementList` is consist of column description `ColumnDef` and `Constraint`. OpenMLDB requires at least one `ColumnDef` in the `TableElementList`.
```
Comment (lumianph) [inline docs/en/reference/sql/ddl/CREATE_TABLE_STATEMENT.md]: `TableElementList` is consist of column description `ColumnDef` and `Constraint`
=>
`TableElementList` consists of `ColumnDef` (column definition) and `Constraint`

Comment (lumianph) [reply docs/en/reference/sql/ddl/CREATE_TABLE_STATEMENT.md]: there is no desc about `Constraint`, which is also inconsistent to the Chinese doc (is it ColumnIndex? which is described in the Chinese doc)

Comment (michelle-qinqin) [reply docs/en/reference/sql/ddl/CREATE_TABLE_STATEMENT.md]: I will change it to `ColumnIndex`

**Review by lumianph — APPROVED** [review]

**Pull request merged**